docs/product-user-guide/src/main/asciidoc/drl-rules-central-create-proc.adoc

```diff
@@ -0,0 +1,187 @@
+[id='drl-rules-central-create-proc']
+= Creating DRL files in {CENTRAL}
+
+You can create and manage DRL files for your project in {CENTRAL}. In each DRL file, you define rule conditions, actions, and other components related to the rule, based on the data objects you create or import in the package.
+
+.Procedure
+. Go to *Menu* -> *Design* -> *Projects* and click the project name.
+. Click *Create New Asset* -> *DRL file*.
+. Enter an informative *DRL file* name and select the appropriate *Package*. The package that you specify must be the same package where the required data objects have been assigned or will be assigned.
```

Comment:
Yeah, I know IBM is also not a fan. I do it a lot though as an economy, and have done the same with DRL file in this example. Otherwise would be "Enter an informative DRL file name in the *DRL file* field and select the appropriate package from the *Package* drop-down menu." Just seems excessive and clunky. I might be alone in this, and can repent if so. One of my deliberate deviations I guess. :)